Platform/Intel/MinPlatformPkg/Acpi/AcpiTables/AcpiPlatform.c

```diff
@@ -18,6 +18,8 @@ typedef struct {
   UINT32   Flags;
   UINT32   SocketNum;
   UINT32   Thread;
+  UINT8    CoreType;
+  BOOLEAN  IsBsp;

```

Comment:
Is "IsBsp" needed? Can we use "Flags"?

Reply:
IsBsp was dropped. Also I kept the original Flags usage since it will be assigned to the Flags in LOCAL_X2APIC_STRUCTURE directly.

Just use a local variable BspIndex to record this info.